vault/src/main/java/org/opencadc/vault/VaultInitAction.java

```diff
@@ -227,14 +240,25 @@ static MultiValuedProperties getConfig() {
     }
 
     static Map<String, Object> getDaoConfig(MultiValuedProperties props) {
-        
         Map<String, Object> ret = new TreeMap<>();
         ret.put(SQLGenerator.class.getName(), SQLGenerator.class); // not configurable right now
-        ret.put("jndiDataSourceName", org.opencadc.vault.VaultInitAction.JNDI_DATASOURCE);
+        ret.put("jndiDataSourceName", org.opencadc.vault.VaultInitAction.JNDI_VOS_DATASOURCE);
         ret.put("schema", props.getFirstPropertyValue(org.opencadc.vault.VaultInitAction.INVENTORY_SCHEMA_KEY));
         ret.put("vosSchema", props.getFirstPropertyValue(org.opencadc.vault.VaultInitAction.VOSPACE_SCHEMA_KEY));
         return ret;
```

Comment:
Does the DaoConfig need both the `vospace` and `inventory` schema's? 
The schema key for both `DaoConfig` and `InvConfig` is the INVENTORY_SCHEMA_KEY.
Also shouldn't need the full class path to those keys, makes the line wrap and hard to decipher when doing a code review. :-)

Reply:
ugh, I think my IDE did that when I pasted... fixing